src/Storage/Service/AbstractUrlStorage.php

```diff
@@ -175,18 +157,13 @@ public function finalizeUpload(string $hash): bool
         try {
             return \copy($source, $destination);
         } catch (\Throwable $e) {
-            $this->logger->info(sprintf('Copy %s to %s failed: %s', $source, $destination, $e->getMessage()));
+            $this->logger->warning(sprintf('Copy %s to %s failed: %s', $source, $destination, $e->getMessage()));
```

Comment:
When using the logger it's better to use the context:

```suggestion
            $this->logger->warning('Copy {source} to {destination} failed: {messag}', [
            $source, 
            $destination, 
            $e->getMessage(),
            $e->getTraceAsString()
            ]);
```